### ISSUE TITLE
[addon: a11y] - rule open in one tab displays as open in another tab when navigating

### ISSUE THREAD

**Issue by CodeByAlex** [storybooks/storybook#6107]
**Describe the bug**
When a rule is set to an open state, where you can see the details about the rule, it remains open in other tabs (pass, violation, incomplete)

**To Reproduce**
1. Set up project
2. Create a component with passing and failing and/or inconsistent rules
 - one rule must be common among at least 2 of the 3 tabs
3. Open the common rule
4. Switch between tabs and notice that the rule remains open

**Expected behavior**
All rules should close when switching between tabs

**Screenshots**
![a11y-open-state-issue](https://user-images.githubusercontent.com/29763590/54427068-cc496180-46ef-11e9-8834-95098977f65c.gif)

**System:**
 - Browser: [chrome, firefox]
 - Framework: [Angular]
 - Version: [5.1.0-alpha.6]

**Additional context**
Found when developing feature #6001

**Comment by Armanio** [comment]
@shilman it actually? i can take this issue?

**Comment by CodeByAlex** [comment]
That would be great @armanio!

**Comment by Armanio** [comment]
Done!

**Comment by shilman** [comment]
Crikey!! I just released https://github.com/storybooks/storybook/releases/tag/v5.1.0-alpha.7 containing PR #6118 that references this issue. Upgrade today to try it out!

Because it's a pre-release you can find it on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.
